app/services/email_service/ses/client.rb

```diff
@@ -0,0 +1,52 @@
+module EmailService::SES
+  class Client
+
+    def initialize(region:, stubs: nil)
+      if region.blank?
+        raise ArgumentError.new("Missing mandatory value for region parameter.")
+      end
+
+      # The SDK v2 automatically gets the AWS_ACCESS_KEY and
+      # AWS_SECRET_ACCESS_KEY from the environment. We only need to
+      # supply region here.
```

Comment:
I guess we could and should use the APP_CONFIG, so that local development settings would be also available